app/helpers/maps_helper.rb

```diff
@@ -1,2 +1,12 @@
 module MapsHelper
+  def data_attributes(map)
+    data = ""
+
+    data += "data-map-background=#{map.background}"
```

Comment:
How about just `data = "data-map-background=#{map.background}"` and remove the `data = ""` above?

Reply:
I don't see why not, if variable `data` is initialized at assignment in js.